tests/Integration/inc/ThirdParty/Hostings/Cloudways/varnishAddonTitle.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\ThirdParty\Hostings\Cloudways;
+
+use WP_Rocket\ThirdParty\Hostings\Cloudways;
+use WPMedia\PHPUnit\Integration\TestCase;
+
+/**
+ * @covers \WP_Rocket\ThirdParty\Hostings\Cloudways::varnish_addon_title
+ * @group Cloudways
+ * @group ThirdParty
+ */
+class Test_VarnishAddonTitle extends TestCase {
+	public function testShouldDisplayVarnishTitleWithCloudways() {
+		$settings = [
+			'varnish_auto_purge' => [
+				'title' => '',
+			],
+		];
+
+		$expected = [
+			'varnish_auto_purge' => [
+				'title' => 'Your site is hosted on Cloudways, we have enabled Varnish auto-purge for compatibility.'
+			],
+		];
```

Comment:
Move to test data fixture. Add more use cases such as:

- An empty settings array is given
- Other settings are given
- If a non-array _could_ be given, pass `false` or `null`.